trackintel/model/tours.py

```diff
@@ -49,13 +55,24 @@ def _validate(obj):
             obj["finished_at"].dtype, pd.DatetimeTZDtype
         ), f"dtype of finished_at is {obj['finished_at'].dtype} but has to be datetime64 and timezone aware"
 
+    @staticmethod
+    def _check(obj):
+        """Check does the same as _validate but returns bool instead of potentially raising an error."""
+        if any([c not in obj.columns for c in _required_columns]):
+            return False
+        if not isinstance(obj["started_at"].dtype, pd.DatetimeTZDtype):
+            return False
+        if not isinstance(obj["finished_at"].dtype, pd.DatetimeTZDtype):
+            return False
+        return True
+
     def to_csv(self, filename, *args, **kwargs):
         """
         Store this collection of tours as a CSV file.
 
         See :func:`trackintel.io.file.write_tours_csv`.
         """
-        ti.io.file.write_tours_csv(self._obj, filename, *args, **kwargs)
+        ti.io.file.write_tours_csv(self, filename, *args, **kwargs)
 
     def to_postgis(
```

Comment:
We need the `@_copy_docstring` here, right?

Reply:
Yes that is true 👍🏼